docs/source/Plugin/_Plugin.rst

```diff
@@ -97,6 +97,9 @@ For example using just like normal task value data:
 * ``[bme#temp.stddevX]`` Compute the standard deviation over the last X samples (or less if there are less samples available)
 * ``[bme#temp.max]`` Refer to the maximum recorded sample since the last ``resetpeaks``. N.B. Not all tasks log the min and max peaks.
 * ``[bme#temp.min]`` See ``[bme#temp.max]`` 
+* ``[bme#temp.size]`` Return the number of samples in memory.
+* ``[bme#temp.sample]`` Access the last sample in memory.
```

Comment:
not last, first (oldest) because last = sample1

Reply:
I did change the behavior as it seemed a bit strange when considering the other use of "N".
If you also need the "oldest", maybe we can also add "first"?

Reply:
Why different? It is different (reverse), from the point of view of the programmer, but not different for the user. This is clearly seen on the graph - the last X is counted from the end. The user is not comfortable counting from the middle. And then sample1, sample2, ... sample(last). Perhaps I initially did not correctly name - the first one is Unix time, but the last one is from the current time )

Reply:
Hmm you might be on to something, like I messed up by using "0" instead of "size() - 1" or something like that.

But in order to grasp it, I really need some coffee (and a lot less interrupts which I'm now dealing with here)...
Will get back to this...

Reply:
N.B. I really appreciate these kinds of "close read" reviews, so keep being critical :)